examples/src/app/index.mjs

```diff
@@ -13,7 +16,4 @@ function main() {
     root.render(jsx(MainLayout, null));
 }
 
-window.onload = () => {
-    // Just a little timeout to give browser some time to "breathe"
-    setTimeout(main, 50);
-};
+main();
```

Comment:
It sounds silly to give it time to breathe, but Chrome would randomly freeze on me during reloads... not sure if this is/was a Linux issue tho lol

Reply:
I have never seen this issue before or could replicate it but let me know if it still persists and Ill look into it 👍